### DISCUSSION TITLE
Baikal CardDav and CalDav server

### DISCUSSION THREAD

**Discussion by lightmaster** [tteck/Proxmox#1094] [Ideas]
Would love to see Baikal get an easy install script. It seems to be one of the top suggested CardDav and CalDav servers. Other CardDav/CalDav server scripts would be cool too, for someone wanting to try several before settling on a single one.

**Comment by mofongox** [comment]
This same repo already has Radicale, if you're interested.

**Reply by lightmaster** [reply]
I don't see Radicale on https://tteck.github.io/Proxmox/
